common/core/desktop/build.sh

```diff
@@ -6,25 +6,37 @@ set -u
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
-. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
-. "$(dirname "$THIS_SCRIPT")/../../../resources/shellHelperFunctions.sh"
+# when building packages we pass in SCRIPTS_DIR because we can't access anything outside of
+# the `desktop` directory
+SCRIPTS_DIR=${SCRIPTS_DIR:-$(dirname "$THIS_SCRIPT")/../../../resources}
+. "${SCRIPTS_DIR}/shellHelperFunctions.sh"
 ## END STANDARD BUILD SCRIPT INCLUDE
 
+THIS_DIR="$(dirname "$THIS_SCRIPT")"
+
+pushd $THIS_DIR > /dev/null
+VERSION=$(./getversion.sh)
+TIER=$(./gettier.sh)
+popd > /dev/null
```

Comment:
This is a real shame 😢  It means Linux package builds are forcing WET build infrastructure for our common code -- which is going to cost us in maintenance in the future. Is this really the case that we cannot access shared resources?

If we do need to go this way, we need to document this very clearly for maintainers and remove the 'STANDARD BUILD SCRIPT INCLUDE' messaging, because it is no longer standard!

Reply:
> Is this really the case that we cannot access shared resources?

For now that's the case.  Since we split the source into separate zip files/source packages for each component it contains only the source code for that component, and the component's directory is the root directory in the zip file we don't have access to anything outside of that directory.

Another motivation to do the Debian source package consolidation (A15S15).